packages/gestalt/src/ModuleExpandable.js

```diff
@@ -23,9 +25,13 @@ export default function ModuleExpandable({
   id,
   accessibilityExpandLabel,
   accessibilityCollapseLabel,
+  expandedId,
+  setExpandedId,
   items,
 }: Props): Node {
-  const [expandedId, setExpandedId] = useState(-1);
+  const [localExpandedId, setLocalExpandedId] = useState(null);
+  const expId = expandedId || localExpandedId;
+  const setExpId = setExpandedId || setLocalExpandedId;
```

Comment:
Let's create a function here, something like `handleChanged`:
```js
const handleChanged = (newExpandedId: ?number) => {
  setExpandedId(newExpandedId);
  if (onExpandedChange) {
    onExpandedChange(newExpandedId);
  }
```

This can be used in place of `setExpId` on line 56.